dolphinscheduler-ui/pnpm-lock.yaml

```diff
@@ -1,125 +1,165 @@
+# with the License.  You may obtain a copy of the License at
 #
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+#   http://www.apache.org/licenses/LICENSE-2.0
 #
-
-lockfileVersion: 5.4
-
-specifiers:
-  '@antv/layout': 0.1.31
-  '@antv/x6': ^1.34.1
-  '@types/js-cookie': ^3.0.2
-  '@types/lodash': ^4.14.185
-  '@types/node': ^18.7.18
-  '@types/nprogress': ^0.2.0
-  '@types/qs': ^6.9.7
-  '@typescript-eslint/eslint-plugin': ^5.37.0
-  '@typescript-eslint/parser': ^5.37.0
-  '@vicons/antd': ^0.12.0
-  '@vitejs/plugin-vue': ^3.1.0
-  '@vitejs/plugin-vue-jsx': ^2.0.1
-  '@vueuse/core': ^9.2.0
-  axios: ^0.27.2
-  dart-sass: ^1.25.0
-  date-fns: ^2.29.3
-  date-fns-tz: ^1.3.7
-  echarts: ^5.3.3
-  eslint: ^8.23.1
-  eslint-config-prettier: ^8.5.0
-  eslint-plugin-prettier: ^4.2.1
-  eslint-plugin-vue: ^9.5.1
-  js-cookie: ^3.0.1
-  lodash: ^4.17.21
-  monaco-editor: ^0.34.0
-  naive-ui: 2.33.5
-  nprogress: ^0.2.0
-  pinia: ^2.0.22
-  pinia-plugin-persistedstate: ^2.2.0
-  prettier: ^2.7.1
-  qs: ^6.11.0
-  sass: ^1.54.9
-  sass-loader: ^13.0.2
-  screenfull: ^6.0.2
-  typescript: ^4.8.3
-  typescript-plugin-css-modules: ^3.4.0
-  vfonts: ^0.0.3
-  vite: ^3.1.2
-  vite-plugin-compression: ^0.5.1
-  vue: ^3.2.39
-  vue-i18n: ^9.2.2
-  vue-router: ^4.1.5
-  vue-tsc: ^0.40.13
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+lockfileVersion: '6.0'
 
 dependencies:
-  '@antv/layout': 0.1.31
-  '@antv/x6': 1.34.1
-  '@vueuse/core': 9.2.0_vue@3.2.39
-  axios: 0.27.2
-  date-fns: 2.29.3
-  date-fns-tz: 1.3.7_date-fns@2.29.3
-  echarts: 5.3.3
-  js-cookie: 3.0.1
-  lodash: 4.17.21
-  monaco-editor: 0.34.0
-  naive-ui: 2.33.5_vue@3.2.39
-  nprogress: 0.2.0
-  pinia: 2.0.22_typescript@4.8.3+vue@3.2.39
-  pinia-plugin-persistedstate: 2.2.0_pinia@2.0.22
-  qs: 6.11.0
-  screenfull: 6.0.2
-  vfonts: 0.0.3
-  vue: 3.2.39
-  vue-i18n: 9.2.2_vue@3.2.39
-  vue-router: 4.1.5_vue@3.2.39
+  '@antv/layout':
```

Comment:
I find this PR update many pnpm lock version, @devosend can you check whether it have to change or not?

Reply:
Lock files can be changed.